pkg/utils/utils.go

```diff
@@ -46,8 +46,8 @@ var runtimeScheme = k8sruntime.NewScheme()
 func init() {
 	//Setting the package-wide version
 	CtrlVersion = os.Getenv("CTRL_VERSION")
-	// For AKO 1.9.1, if ctrl version is 22.1.3, make it 22.1.2
-	if CtrlVersion == CTRL_VERSION_22_1_3 {
+	// For AKO 1.9.x, if ctrl version is >= 22.1.3, make it 22.1.2
+	if CtrlVersion >= CTRL_VERSION_22_1_3 {
```

Comment:
@arihantg: Similar change has to be done for: https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/release-1.9.3/pkg/utils/avi_rest_utils.go#L104

Reply:
Update there as well